Upgraded dependencies

Razican opened the pull request:
Hello,
I upgraded the dependencies of the crate, checked that everything worked as expected, and updated the version number to 0.11.0.

Let me know if that version bump is not needed (since the API does not change).

Razican committed:
Upgraded dependencies

 - cgmath 0.16 => 0.17
 - lazy_static 1.0 => 1.3
 - serde_cbor 0.8 => 0.9

This will upgrade the num_traits sub-dependency from 0.1 to 0.2.

Razican committed:
Updated version number

TimonPost commented:
Ehmmm, I have no clue why github showed me this notification, did you refer to me somewhere?

TimonPost commented on Cargo.toml:
```diff
@@ -8,16 +8,16 @@
 	readme = "README.md"
 	repository = "https://github.com/omarkmu/winconsole"
-	version = "0.10.0"
+	version = "0.11.0"
```
I guess, since I am here, I can directly do a review, are there breaking changes? I don't see them in this PR, as you said as well, so why not doing a minor version upgrade like 0.10.1? That would be perfectly fine.

Razican replied:
True, I changed the version.

Razican committed:
Correct version number

The correct version number should be 0.10.1, since there were no breaking changes.

Razican commented:
> 
> 
> Ehmmm, I have no clue why github showed me this notification, did you refer to me somewhere?

I would guess it's because you're watching the repository.

TimonPost commented:
Ah... forgot that I did that, my bad :) I already know why I did it btw.

mati865 commented:
@omarkmu has been missing for over 6 months now.

omarkmu commented:
I haven't been around, sorry. Taking a look at this now.

Razican commented:
Hi! thanks for the merge :) would it be possible to release it on crates.io?
